packages/widgetbook/lib/src/knobs/knobs_panel.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:widgetbook/src/knobs/knobs.dart';
+
+/// The panel containing the knobs for a usecase
+class KnobsPanel extends StatelessWidget {
+  const KnobsPanel({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final knobs = context.watch<KnobsNotifier>().all();
+    final noKnobs = knobs.isEmpty;
+    final title = noKnobs ? 'No knobs to dispaly' : 'Knobs';
+    return SingleChildScrollView(
+      child: Column(
+        children: [
+          Text(
+            title,
+            style: const TextStyle(
+              fontSize: 20,
+            ),
+          ),
+          ...knobs.map((e) => e.build())
+        ],
+      ),
```

Comment:
I think the `Column` should use `CrossAxisAlignment.start`. The Property Panel does not have a title which makes this design diverge from what has already been defined. However, I see that we need some sort of text that is displayed whenever no knobs are defined for a use case.